tableaudocumentapi/workbook.py

```diff
@@ -11,25 +11,23 @@
 
 
 class Workbook(object):
-    """
-    A class for writing Tableau workbook files.
-
-    """
+    """A class for writing Tableau workbook files."""
 
     ###########################################################################
```

Comment:
I left the public and private api bits in, I personally am not a fan, so I'm happy to remove them.

But I was deferring to your original commit 🍭